### PR TITLE
feat: add question issue template

### PR DESCRIPTION
**Pull request by tzssangglass**
### What this PR does / why we need it:

add question issue template

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Questions
+about: Questions and answers indirectly related to APISIX
```
Comment (spacewander) [inline .github/ISSUE_TEMPLATE/question.md:3]: Why use "indirectly"?

Comment (tzssangglass) [reply .github/ISSUE_TEMPLATE/question.md:3]: I was trying to express that these questions are not caused by APISIX, but may be encountered in the course of using APISIX.

Comment (spacewander) [reply .github/ISSUE_TEMPLATE/question.md:3]: Maybe we can remove it to make this less confusing? It's hard to distinguish the borderline between directly and indirectly.

Comment (tzssangglass) [reply .github/ISSUE_TEMPLATE/question.md:3]: got

Comment (tzssangglass) [reply .github/ISSUE_TEMPLATE/question.md]: update

**Pull request closed without merging**